src/lib/components/board-member/board-picture.svelte

```diff
@@ -15,7 +16,11 @@
       />
     </clipPath>
   </defs>
-  <image clip-path="url(#shape)" xlink:href={src} />
+  <foreignObject x="0" y="0" width="100%" height="100%" clip-path="url(#shape)">
+    <div xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
I'd recommend resolving the lint errors. [Here is one](https://github.com/EthanThatOneKid/acmcsuf.com/actions/runs/6089638664/job/16522854449#step:7:55):

```
/home/runner/work/acmcsuf.com/acmcsuf.com/src/lib/components/board-member/board-picture.svelte:20:10
Error: Argument of type '{ xmlns: string; }' is not assignable to parameter of type 'HTMLProps<"div", HTMLAttributes<any>>'.
  Object literal may only specify known properties, and '"xmlns"' does not exist in type 'HTMLProps<"div", HTMLAttributes<any>>'. (ts)
  <foreignObject x="0" y="0" width="100%" height="100%" clip-path="url(#shape)">
    <div xmlns="http://www.w3.org/1999/xhtml">
      <img {src} alt="Board member" class:legacy />


====================================
svelte-check found 1 error and 0 warnings in 1 file

Error: Process completed with exit code 1.
```

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/899/commits/5a686d7522a4853baf2b49a001877295ff3bc12e>.